From 5c467c705c9e875e8a18cb26b47cc8e601110748 Mon Sep 17 00:00:00 2001
From: WANG Xuerui <git@xen0n.name>
Date: Mon, 9 Aug 2021 21:53:46 +0800
Subject: [PATCH 11/11] LoongArch: make .text start at 0x10000

The previous value obviously is coming from MIPS; no reason is given as
to why this is absolutely needed. Just change to be more like RISC-V and
a few other arches.
---
 ld/emulparams/elf64loongarch-defs.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ld/emulparams/elf64loongarch-defs.sh b/ld/emulparams/elf64loongarch-defs.sh
index ed793565a06..7490f752afb 100644
--- a/ld/emulparams/elf64loongarch-defs.sh
+++ b/ld/emulparams/elf64loongarch-defs.sh
@@ -29,7 +29,7 @@ esac
 
 NOP=0x03400000
 
-TEXT_START_ADDR=0x120000000
+TEXT_START_ADDR=0x10000
 MAXPAGESIZE="CONSTANT (MAXPAGESIZE)"
 COMMONPAGESIZE="CONSTANT (COMMONPAGESIZE)"
 
-- 
2.33.0

